scripts/install-mysql-minio.sh

```diff
@@ -4,13 +4,9 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 ## Create the mysql operator
 # Apply Namespace if already exists
-cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: mysql-operator
-EOF
-kubectl apply -n mysql-operator -f "${DIR}"/../docs/mysql-operator/mysql-operator.yaml
+helm repo add bitpoke https://helm-charts.bitpoke.io
+helm repo update 
+helm install mysql-operator bitpoke/mysql-operator --namespace mysql-operator --create-namespace --set "extraArgs={--mysql-versions-to-image=5.7.26=percona:5.7.35}" --version v0.6.2
```

Comment:
Is this correct, since we are using MysqlVersion 8.0?

Reply:
@toninis any ideas here?

Reply:
It's irrelevant . This fix is for the MySQL 5.7 version . We can leave this for backwards compatibility if we ever want to test against mysql 5.7